idea-valley/Kandidatas.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+class Kandidatas extends IdeaValley
+{
+    public function run($kandidatas)
+    {
+        $sugebejimai = array(
+            'PHP 5.5',
+            'PHP 5.6',
+            'MySQL',
+            'ORM', //(Eloquent būtų privalumas);
+            'PHP Framework', //(Laravel būtų privalumas);
+            'MVC',
+            'GIT',
+            'HTML5',
+            'CSS3',
+            'JQuery'
```

Comment:
Be aware of jQuery becoming your main dependency for everything :) It is **NOT FRAMEWORK**!